src/mame/includes/p2000t.h

```diff
@@ -17,13 +17,22 @@
 #include "machine/p2000t_mdcr.h"
 #include "machine/ram.h"
 #include "emupal.h"
+#include "screen.h"
 
+#define LUT_TABLE_SIZE 16 
+
+#define Z80PIO_MODE_OUTPUT 0x00
+#define Z80PIO_MODE_INPUT  0x01
+#define Z80PIO_MODE_BIDIR  0x02
+#define Z80PIO_MODE_CTRL   0x03
```

Comment:
You shouldn’t be leaking preprocessor macros from a header.  Also, if the machine uses Zilog PIO devices, you should use MAME’s device emulation, not bloat the driver with another implementation of it.